test/sql/updates/setup.continuous_aggs.v2.sql

```diff
@@ -70,4 +70,55 @@ AS
   GROUP BY bucket, location
   HAVING min(location) >= 'NYC' and avg(temperature) > 2;
 
-  REFRESH MATERIALIZED VIEW mat_before;
+REFRESH MATERIALIZED VIEW mat_before;
+
+-- we create separate schema for realtime agg since we dump all view definitions in public schema
+-- but realtime agg view definition is not stable across versions
+CREATE SCHEMA cagg;
+
+CREATE VIEW cagg.realtime_mat
+WITH ( timescaledb.continuous, timescaledb.materialized_only=true, timescaledb.refresh_lag='-30 day', timescaledb.max_interval_per_job ='1000 day')
```

Comment:
shouldn't this be materialized_only = false?

Reply:
good catch. fixed